packages/components/toggle/src/__tests__/Toggle.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import Toggle from '../Toggle';
+import { ThemeProvider } from '@douro-ui/react';
+
+describe('<Toggle />', () => {
+  it('renders with toggle checked', () => {
+    render(
+      <ThemeProvider>
+        <Toggle checked={true}>This is a toggle</Toggle>
+      </ThemeProvider>,
+    );
+
+    const toggleElement = screen.getByRole('checkbox', { checked: true });
+
+    expect(toggleElement).toBeInTheDocument();
+  });
+
+  it('renders with toggle not checked', () => {
+    render(
+      <ThemeProvider>
+        <Toggle>This is a toggle</Toggle>
+      </ThemeProvider>,
+    );
+
+    const toggleElement = screen.getByRole('checkbox', { checked: false });
+
+    expect(toggleElement).toBeInTheDocument();
+  });
```

Comment:
Those tests are not testing the component properly, you need to make sure the component is checked or unchecked as well